clt_tests/tests/test-backup-restore.rec

```diff
@@ -0,0 +1,210 @@
+| rt_without_columnar | CREATE TABLE rt_without_columnar (
+id bigint,
+title text,
+category_id integer,
+price float,
+description string attribute,
+tags multi,
+attributes json
+) |
++---------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+––– input –––
+docker exec manticore mysql -e "show create table test"
+––– output –––
++-------+--------------------------------------------------------------------------------------------------------------------------+
+| Table | Create Table                                                                                                             |
++-------+--------------------------------------------------------------------------------------------------------------------------+
+| test  | CREATE TABLE test (
+id bigint,
+title text,
+image_vector float_vector knn_type='hnsw' knn_dims='4' hnsw_similarity='L2'
+) |
++-------+--------------------------------------------------------------------------------------------------------------------------+
+––– input –––
+docker exec -it manticore manticore-backup --backup-dir=/tmp
+––– output –––
+Copyright (c) %{YEAR}-%{YEAR}, Manticore Software LTD (https://manticoresearch.com)
+Manticore config file: /etc/manticoresearch/manticore.conf.sh
+Tables to backup: all tables
+Backup dir: /tmp
+Manticore config
+endpoint =  http://127.0.0.1:9308
+Manticore config
+endpoint =  http://127.0.0.1:9312
+Manticore versions:
+manticore: %{SEMVER} %{COMMITDATE} dev
+columnar: %{SEMVER} %{COMMITDATE}
+secondary: %{SEMVER} %{COMMITDATE}
+knn: %{SEMVER} %{COMMITDATE}
+buddy: %{SEMVER}
+[93m%{DATETIME}[0m [Info] Starting the backup...
+[93m%{DATETIME}[0m [Info] Backing up config files...
+[93m%{DATETIME}[0m [Info]   config files - [92mOK[0m
+[93m%{DATETIME}[0m [Info] Backing up global state files...
+[93m%{DATETIME}[0m [Info]   global state files – [92mOK[0m
+[93m%{DATETIME}[0m [Info] Backing up tables...
+[93m%{DATETIME}[0m [Info]   distributed_index (distributed)...
+[93m%{DATETIME}[0m [Info]   [93mSKIP[0m
+[93m%{DATETIME}[0m [Info]   rt_with_columnar (rt) [#!/[0-9]{1}\.[0-9]{3}/!#K]...
+[93m%{DATETIME}[0m [Info]    [92mOK[0m
+[93m%{DATETIME}[0m [Info]   rt_without_columnar (rt) [#!/[0-9]{1}\.[0-9]{3}/!#K]...
+[93m%{DATETIME}[0m [Info]    [92mOK[0m
+[93m%{DATETIME}[0m [Info]   test (rt) [%{NUMBER}B]...
+[93m%{DATETIME}[0m [Info]    [92mOK[0m
+[93m%{DATETIME}[0m [Info] Running sync
+[93m%{DATETIME}[0m [Info]  [92mOK[0m
+[93m%{DATETIME}[0m [Info] You can find backup here: /tmp/backup-%{NUMBER}
+[93m%{DATETIME}[0m [Info] Elapsed time: #!/([0-9]+\.[0-9]+|0)/!#s
+[93m%{DATETIME}[0m [Info] Done
+––– input –––
+BACKUP_NAME=$(docker exec manticore ls /tmp/ | grep backup-*)
```

Comment:
This variable should be exposed between commands. I'm pretty sure that below it equals empty line

Reply:
No, it's working correctly here.
```
––– input –––
BACKUP_NAME=$(docker exec manticore ls /tmp/ | grep backup-*)
––– output –––
––– input –––
echo $BACKUP_NAME
––– output –––
backup-20240717174115
```